src/index.js

```diff
@@ -3,18 +3,21 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
-import { BrowserRouter } from 'react-router-dom';
-import { CartProvider, ContextProvider } from "./CartContext/CartContext";
+import { BrowserRouter } from "react-router-dom";
+import { CartProvider } from "./Context/CartContext";
+import { FilterProvider } from "./Context/FilterContext";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
-      <CartProvider>
-        <App />
-      </CartProvider>
+      <FilterProvider>
```

Comment:
suggest: can make one index file for context Provider pass children Between all providers All App.js looks clean